pom.xml

```diff
@@ -168,33 +168,55 @@
 			<scope>test</scope>
 		</dependency>
 
-		<!--  Dependencies used by JSON Convert to manipulate JSONB -->
+		<!--  Dependencies used by JSON Convert to manipulate (JSON-B) -->
 		<dependency>
 			<groupId>javax.json.bind</groupId>
 			<artifactId>javax.json.bind-api</artifactId>
 			<version>1.0</version>
 		</dependency>
-		<!--  https://projects.eclipse.org/projects/ee4j.yasson -->
+		<!--  https://projects.eclipse.org/projects/ee4j.yasson (JSON-B) -->
 		<dependency>
 			<groupId>org.eclipse</groupId>
 			<artifactId>yasson</artifactId>
-			<version>1.0</version>
+			<version>1.0.7</version>
 		</dependency>
-
-		<!-- Dependency used to manipulate JSON format -->
+		<!-- Dependency used to manipulate JSON format (JSON-B and JSON-P) -->
 		<dependency>
 			<groupId>org.glassfish</groupId>
 			<artifactId>javax.json</artifactId>
 			<version>1.1.4</version>
 			<scope>runtime</scope>
 		</dependency>
 		
-		<!-- Dependency used to manipulate JSON format -->
+		<!-- Dependency used to manipulate JSON format (JSON-P) -->
 		<dependency>
 			<groupId>javax.json</groupId>
 			<artifactId>javax.json-api</artifactId>
-			<version>1.1</version>
+			<version>1.1.4</version>
+		</dependency>
+		
+		<!-- JAX-RS dependency for establishing HTTP connections -->
+		<dependency>
+			<groupId>jakarta.ws.rs</groupId>
+			<artifactId>jakarta.ws.rs-api</artifactId>
+			<version>2.1.6</version>
 		</dependency>
+		<dependency>
+			<groupId>org.glassfish.jersey.core</groupId>
+			<artifactId>jersey-client</artifactId>
+			<version>2.30.1</version>
```

Comment:
runtime only (et revoir le reste)

Reply:
Fix in a80a8f27fd1a55806ab72635368de4f7dac3f16d